modules/images-create-s2i-scripts.adoc

```diff
@@ -120,31 +123,20 @@ EOF
 ----
 
 
-////
-== Using Images with ONBUILD Instructions
-The `ONBUILD` instructions can be found in many official container images. For
-example:
+[id="images-s2i-requirements-using-images-with-onbuild-instructions_{context}"]
+== Using images with ONBUILD instructions
```

Comment:
I think we can drop this section entirely. ONBUILD instructions should work to the extent that they are supported in Buildah.